uint/Cargo.toml

```diff
@@ -34,7 +34,7 @@ required-features = ["std"]
 criterion = "0.3.0"
 num-bigint = "0.3.1"
 
-[target.'cfg(unix)'.dev-dependencies]
+[target.'cfg(all(unix, target_arch = "x86_64"))'.dev-dependencies]
```

Comment:
reason behind this change? Does rug only work on LE?

Reply:
no, but it can't cross compile to `mips64-unknown-linux-gnuabi64`
that's fine, it's only used for benches and fuzz tests